ui/css/perf.css

```diff
@@ -268,6 +268,30 @@ th {
   color: #333;
 }
 
+.compare-table .subtest-header-visible .sorted-asc {
+  border-bottom-color: #333;
+  border-bottom-width: 1px;
+}
+
+.compare-table  .subtest-header-visible .sorted-desc {
+  border-top-color: #333;
+  border-top-width: 1px;
+}
+
+.compare-table .subtest-header-visible > th,
+.compare-table .subtest-header-visible a,
+.compare-table .subtest-header-visible button {
```

Comment:
I tried just `.compare-table .subtest-header-visible` but after rebasing didn't work anymore. Any suggestions how would I define this in a simpler form than `.subtest-header` is defined?

Reply:
Maybe give each of those elements a className instead of basing it on inheritance? For ex:
```
my-special-class {
  color: #333;
  transition: None;
}
```

Otherwise you can use`!important` if you have to.